### PR TITLE
Adjust upload yaml

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- Use sap ui5 components in upload yaml right view

**Related issue(s)**

https://github.com/kyma-project/busola/issues/2872

### CODE REVIEW

**Review by dbadura — COMMENTED** [review]
In other PR I will adjust Monaco scroll, scroll of the right side and upload finished view when I get the desings.

File: public/defaultConfig.yaml [hunk 1]
```diff
@@ -72,7 +72,7 @@ config:
         policies:
           - Default
     CLUSTER_VALIDATION:
-      isEnabled: false
+      isEnabled: true
```
Comment (akucharska) [inline public/defaultConfig.yaml:75]: Should we set it to true?

**Review by akucharska — CHANGES_REQUESTED** [review]
1. Is the font right size? 
Screenshot 2024-04-23 at 08 38 13
2. Is there a way to fix separator lines
3. The warning icon is not centered to the arrow of collapsible row
Screenshot 2024-04-23 at 08 40 35
4. There should be bigger space between title and text above it 
Screenshot 2024-04-23 at 08 40 41

File: public/defaultConfig.yaml (hunk 1 above)
Comment (dbadura) [reply public/defaultConfig.yaml:75]: Reverted

File: src/resources/Namespaces/YamlUpload/ResourceValidationResult.js [hunk 2]
```diff
@@ -79,22 +79,31 @@ const ValidationWarnings = ({ resource, validationSchema }) => {
...
       {warnings.flat().map(warning => (
         <>
-          <FlexBox>
+          <FlexBox alignItems={'Begin'}>
             <ObjectStatus
               showDefaultIcon
               state={ValueState.Warning}
-              style={spacing.sapUiSmallMarginEnd}
+              style={{
+                marginLeft: '-0.3125rem', //set icon in one line with expand arrow. The value from class `--_ui5-v1-20-0_panel_content_padding1` is divided by 2
```
[the remark this answers is not in the document]
Comment (dbadura) [reply src/resources/Namespaces/YamlUpload/ResourceValidationResult.js:87]: Unfortunately I couldn't make it to work.

**Pull request merged**